Analytics-API/MQTT.md

```diff
@@ -0,0 +1,116 @@
+---
+id: mqtt-integration
```

Comment:
Unless there is clear need, it's probably neatest to not specify id's and use the [default behaviour (document name)](https://v2.docusaurus.io/docs/2.0.0-alpha.69/docs-introduction#document-id).

Reply:
I had used this field to override the id being the path from docs/ (in this instance it would be analytics-api/mqtt) as the page "name" would be more intuitive imo. Valid if you want to stick with the default. I can see good arguments like forced consistency - having IDs be automatic means they are guaranteed to be in a predictable format.

Reply:
In that case, can the file simply be renamed to `mqtt-integration` to create that consistency?

Reply:
Yep, I think that's the best bet.